DEPLOYMENT.md

```diff
@@ -0,0 +1,29 @@
+# Deployment of EvalAI-ngx
+
+To deploy the Project you need to have nodejs & npm installed.
+
+1. Get the source code on to your machine via git.
+
+    ```shell
+    git clone https://github.com/Cloud-CV/EvalAI-ngx.git evalai-ngx && cd evalai-ngx
+    ```
+
+2. Install all the Project Requirements with npm.
+
+    ```
+    npm install
+    ```
+
+3. Build the project with ng. The Artifacts will be stored in the `dist/` directory.
```

Comment:
Change this to `Build the project with ng. The artifacts will be stored in the `dist/` directory.`